### PR TITLE
Remove root node fetch in placeArrowBottomLeft

### PR DESCRIPTION
**Pull request by zhx828**
Cannot locate the `this` reference which causes page goes blank. The targetEl is not used anywhere.

Not sure which recent change causes this issue. Maybe a better solution exists with more knowledge from @onursumer @alisman 

Jan-11-2022 13-41-37

### CODE REVIEW

**Comment by zhx828** [comment]
I only put it as draft PR waiting the confirmation on the fix. Happy to convert to regular once getting the team's approval.

**Comment by onursumer** [comment]
It's not clear to me why we have an unused `targetEl`. Maybe we were using it previously but not anymore? If removing it fixes the issue I would say let's do it.

**Review by onursumer — APPROVED** [review]
It should be okay to remove an unused variable

**Comment by inodb** [comment]
Duplicate of https://github.com/cBioPortal/cbioportal-frontend/pull/4118

**Pull request closed without merging**
